src/configs/configureMobX.ts

```diff
@@ -0,0 +1,11 @@
+export const config = {
+  useProxies: 'ifavailable',
+  computedRequiresReaction: true,
+  reactionRequiresObservable: true,
+  observableRequiresReaction: true,
+} as {
+  useProxies: 'ifavailable' | 'always' | 'never' | undefined;
+  computedRequiresReaction: boolean | undefined;
+  reactionRequiresObservable: boolean | undefined;
+  observableRequiresReaction: boolean | undefined;
+};
```

Comment:
А mobx не предоставляет тип для конфига?
Если нет, то может вынести в отдельный тип
```suggestion
type MobxConfig = {
  useProxies: 'ifavailable' | 'always' | 'never' | undefined;
  computedRequiresReaction: boolean | undefined;
  reactionRequiresObservable: boolean | undefined;
  observableRequiresReaction: boolean | undefined;
};

export const config: MobxConfig = {
  useProxies: 'ifavailable',
  computedRequiresReaction: true,
  reactionRequiresObservable: true,
  observableRequiresReaction: true,
};
```